src/xrGame/quadtree.h

```diff
@@ -49,7 +49,10 @@ class CQuadTree
         virtual ~CFixedStorage() { xr_free(m_objects); }
         IC T* get_object()
         {
-            VERIFY(m_free);
+            if (m_free == NULL)
+            {
+              return NULL;
+            }
```

Comment:
Codestyle, plus it's better to leave VERIFY here, because it's not normal when m_free is null.
```suggestion
            VERIFY(m_free);
            if (!m_free)
                return nullptr;

```

Reply:
Fixed

Reply:
Partially fixed, but not fully because:

1. We always use nullptr, NULL is not used (and is replaced to nullptr during code refactoring)
2. We don't check for nullptr explicitly, where it's possible to not do it.

Reply:
> Partially fixed, but not fully because:
> 
>     1. We always use nullptr, NULL is not used (and is replaced to nullptr during code refactoring)
> 
>     2. We don't check for nullptr explicitly, where it's possible to not do it.

Well there is need to check if we have nullptr, otherwise there is a crash.

Reply:
> > Partially fixed, but not fully because:
> > ```
> > 1. We always use nullptr, NULL is not used (and is replaced to nullptr during code refactoring)
> > 
> > 2. We don't check for nullptr explicitly, where it's possible to not do it.
> > ```
> 
> Well there is need to check if we have nullptr, otherwise there is a crash.

I mean, we use `if (!ptr)` of `if (ptr)` checks rather than directly comparing it to nullptr.